apps/full-site-editing/full-site-editing-plugin/starter-page-templates/class-starter-page-templates.php

```diff
@@ -172,15 +172,16 @@ public function enqueue_assets() {
 		$config = apply_filters(
 			'fse_starter_page_templates_config',
 			[
-				'siteInformation' => array_merge( $default_info, $site_info ),
-				'templates'       => array_merge( $default_templates, $vertical_templates ),
-				'vertical'        => $vertical,
-				'segment'         => $segment,
+				'siteInformation' 		=> array_merge( $default_info, $site_info ),
+				'templates'       		=> array_merge( $default_templates, $vertical_templates ),
+				'vertical'        		=> $vertical,
+				'segment'         		=> $segment,
 				// phpcs:ignore WordPress.Security.NonceVerification.NoNonceVerification
-				'screenAction'    => isset( $_GET['new-homepage'] ) ? 'add' : $screen->action,
-				'theme'           => normalize_theme_slug( get_stylesheet() ),
+				'screenAction'    		=> isset( $_GET['new-homepage'] ) ? 'add' : $screen->action,
+				'theme'           		=> normalize_theme_slug( get_stylesheet() ),
 				// phpcs:ignore WordPress.Security.NonceVerification.NoNonceVerification
-				'isFrontPage'     => isset( $_GET['post'] ) && get_option( 'page_on_front' ) === $_GET['post'],
+				'isFrontPage'     		=> isset( $_GET['post'] ) && get_option( 'page_on_front' ) === $_GET['post'],
+				'hideFrontPageTitle'	=> get_theme_mod( 'hide_front_page_title' ),
```

Comment:
I wanted to suggest a default value of false here but it turns out that `false` is a default anyway right in `get_theme_mod` 😀 No need to do anything then

```suggestion
				'hideFrontPageTitle'	=> get_theme_mod( 'hide_front_page_title', false ),
```